pkgs/generate.sh

```diff
@@ -0,0 +1,30 @@
+
+#!/bin/sh
+
+function runInShell {
+  nix-shell -j 4 -p cabal2nix nix-prefetch-scripts coreutils cabal-install stack --run "$*"
+}
+
+set -xe
+set -v
+
+# Get relative path to script directory
+scriptDir=$(dirname -- "$(readlink -f -- "${BASH_SOURCE[0]}")")
+
+source ${scriptDir}/../scripts/set_nixpath.sh
```

Comment:
I think you'd have to quote all uses of `$scriptDir`, as in the above line and the below lines, as well.